addons/event_stand/models/event_stand.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class EventStand(models.Model):
```

Comment:
If I remember well, this is the "main" object people use from event. In that case, shouldn't we have a chatter on it ? We could log update for slots being booked and/or confirmed on it. Maybe to check with LNA. Currently I fear that there is few information given to event managers and it would become complicated to follow on a big event...